src/interactive.js

```diff
@@ -0,0 +1,155 @@
+/*eslint-disable*/
+/* eslint-disable no-plusplus */
+const clearBtn = document.querySelector('.clear-completed');
+
+let todos = JSON.parse(localStorage.getItem('todo-list')) || [];
```

Comment:
- [ ] Kindly, avoid disabling Eslint for all your code, Eslint is one of the basic requirements of good programming skills. It helps you to analyze your code to check construct, syntax, and avoidable bugs. Please, remove it from all your code to ensure that some of your subprograms or functions pass the linter check.